constants/mock-data/mockData.tsx

```diff
@@ -123,13 +123,23 @@ export const mockTransaction = [
   },
 ];
 
-export const mockMarketData1 = 
-  {
-    fileHash: 'demo1',
-    name: 'Minions 6',
-    size: 3000,
-    id: '1',
-    ip: '192.168.1.101',
-    port: 8080,
-    price: 0.05,
-  };
\ No newline at end of file
+export const mockMarketData1 = {
+  fileHash: 'demo1',
+  name: 'Minions 6',
+  size: 3000,
+  id: '1',
+  ip: '192.168.1.101',
+  port: 8080,
+  price: 0.05,
+};
+
+export const mockStatsMonthData = [
```

Comment:
Nothing major, just a nitpick about the types of this list. We could create a typing for the Stat section such as 

```
export type Stats= {
     label: string;
     value: number;
}
```